build_tools/build_detect_platform

```diff
@@ -66,11 +66,19 @@ rm -f "$OUTPUT"
 touch "$OUTPUT"
 
 if test -z "$CC"; then
-   CC=cc
+    if [ -x "$(command -v clang)" ]; then
+        CC=clang
```

Comment:
Just wanted to point out that this could cause a sudden change of behavior on systems where clang and gcc are already installed, and cc was being used before by default.

Reply:
Okay, I can revise this. So if gcc is present, you would like that to be the default?

Reply:
yeah, to preserve the existing behavior.